lib/jekyll/commands/new.rb

```diff
@@ -137,11 +137,7 @@ def bundle_install(path)
           Jekyll::External.require_with_graceful_fail "bundler"
           Jekyll.logger.info "Running bundle install in #{path.cyan}..."
           Dir.chdir(path) do
-            if ENV["CI"]
-              system("bundle", "install", "--quiet")
-            else
-              system("bundle", "install")
-            end
+            Jekyll.logger.info `bundle install`
```

Comment:
Nope, it runs the command and pipes its output to `logger`.

It doesn't just print `bundle install` 😂